src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -13,5 +17,11 @@ public class FeatureManagementOptions
         /// If missing feature filters are not ignored an exception will be thrown when attempting to evaluate a feature that depends on a missing feature filter.
         /// </summary>
         public bool IgnoreMissingFeatureFilters { get; set; }
+
+        /// <summary>
+        /// Enables notification of feature flags that are missing a configuration.
+        /// If a feature flag exists with no configuration then this delegate will be invoked.
+        /// </summary>
+        public Func<string, Task> OnMissingFeatureConfiguration { get; set; }
```

Comment:
How about just `OnMissingFeature`?

The Func should return `Task<bool>`, so someone can actually use this and change the default behavior (for example, return true. BTW, we should have a test for this scenario.).

Reply:
The name `OnMissingFeature` sounds good to me.

What would the boolean indicate? Whether or not to error? My initial thoughts were you could use this delegate to choose how a missing feature is handled. For instance I could choose to throw an exception within the delegate if I so wished.

Reply:
Sorry for my brevity in the previous comment. Currently, the feature is considered false by default. Returning `Task<bool>` will enable full customization of the default behavior. For example, someone can change to always consider a feature is 'On' if it's missing. Another one can even have more complex logic and decide dynamically whether a feature should be considered on or off if it's missing.

Reply:
Great, thanks for the clarification, see my latest commit.